itest/lib/brim-driver.ts

```diff
@@ -0,0 +1,151 @@
+import {Application} from "spectron"
```

Comment:
This file is almost a copy of "createTestBrim". I'll remove the old version if this design gets approval.